src/app/Scenes/MyCollection/Screens/Artwork/Components/ArtworkInsights/MyCollectionArtworkComparableWorks.tsx

```diff
@@ -0,0 +1,88 @@
+import { ActionType, ContextModule, OwnerType } from "@artsy/cohesion"
+import { MyCollectionArtworkComparableWorks_artwork$key } from "__generated__/MyCollectionArtworkComparableWorks_artwork.graphql"
+import { AuctionResultListItemFragmentContainer } from "app/Components/Lists/AuctionResultListItem"
+import { SectionTitle } from "app/Components/SectionTitle"
+import { navigate } from "app/navigation/navigate"
+import { extractNodes } from "app/utils/extractNodes"
+import { useScreenDimensions } from "app/utils/useScreenDimensions"
+import { Flex, Separator } from "palette"
+import React from "react"
+import { FlatList } from "react-native"
+import { graphql, useFragment } from "react-relay"
+import { useTracking } from "react-tracking"
+
+interface MyCollectionArtworkComparableWorksProps {
+  artwork: MyCollectionArtworkComparableWorks_artwork$key
+}
+
+export const MyCollectionArtworkComparableWorks: React.FC<
+  MyCollectionArtworkComparableWorksProps
+> = (props) => {
+  const { trackEvent } = useTracking()
+
+  const artwork = useFragment<MyCollectionArtworkComparableWorks_artwork$key>(
+    artworkFragment,
+    props.artwork
+  )
+
+  const comparableWorks = extractNodes(artwork?.comparableAuctionResults)
+
+  if (!comparableWorks.length) {
+    return null
+  }
+
+  return (
+    <Flex mb={6}>
+      <SectionTitle
+        title="Comparable Works"
+        onPress={() => {
+          trackEvent(tracks.tappedShowMore(artwork?.internalID, artwork?.slug))
+          navigate(`/artist/${artwork?.artist?.slug!}/auction-results`)
+        }}
+      />
+
+      <FlatList
+        data={comparableWorks}
```

Comment:
```suggestion
      <FlatList
        listKey="comparable-works"
        data={comparableWorks}
```
**non-blocking**: When I was testing the page, I saw this error about the parent scrollview having two flatlists as children, which hinders the virtualization from work properly:

![image](https://user-images.githubusercontent.com/20655703/156557417-93bb07c8-e259-4d46-a89b-b2b512cbd4d4.png)

this suggestion is just to stop the error from showing, but I don't believe it necessary since we have a limited number of items for the two flatlists

Reply:
Good catch!